spotty/spotty/Data/Models/UserProfile.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UserProfile.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 13.06.22.
+//
+
+import Foundation
+
+struct UserProfile: Codable {
+
+    let country: String
+    let displayName: String
+    let email: String
+    let externalURLs: [String: String]
+    let id: String
+    let product: String
+    let images: [SpotifyImage]
+
+    private enum CodingKeys: String, CodingKey {
+        case country = "country"
```

Comment:
You can delete rawValues in all cases when they are same as the case itself.

Reply:
done